docs/build/guides/dapps/working-with-contract-specs.mdx

```diff
@@ -327,16 +327,21 @@ def send_transaction() -> soroban_rpc.SendTransactionResponse:
     root_keypair = Keypair.from_secret(
         "SXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
     )
-    root_account = server.load_account("GBSBL6FBPX5UHKL4AZCPUU6PXKUBYMKRUN3L4YQ4V2CCWSE7YMN2HYPB")
+    root_account = Account(account=root_keypair.public_key, sequence=1)
```

Comment:
Is there a reason to create our own account `Account()` object here? the sequence will be greater than `1`, so this transaction wouldn't actually be valid. typically the `server.load_account()` function is the way i've done it so you're sure to have the correct sequence number for the next transaction